en/notes-tidb-operator-v1.1.md

```diff
@@ -0,0 +1,111 @@
+
+### Initializer
+
+- If the initialization job is executed before TiDB Operator is upgraded to v1.1, the initialization job does not need to be migrated from tidb-cluster chart to TidbInitializer CR.
+
+- If the initialization job is not executed before TiDB Operator is upgraded to v1.1, and the password for the TiDB root user has not been modified, you need to initialize your cluster after upgrading to TiDB Operator v1.1. For details, refer to [Initialize a TiDB Cluster in Kubernetes](initialize-a-cluster.md).
+
+### Pump
+
+After TiDB Operator is upgraded to v1.1, you can modify the TidbCluster CR and add the configuration of Pump. You can then manage the Pump component using TidbCluster CR.
+
+``` yaml
+spec
+  ...
+  pump:
+    baseImage: pingcap/tidb-binlog
+    version: v3.0.11
+    replicas: 1
+    storageClassName: local-storage
+    requests:
+      storage: 30Gi
+    schedulerName: default-scheduler
+    config:
+      addr: 0.0.0.0:8250
+      gc: 7
+      heartbeat-interval: 2
+```
+
+You can modify `version`, `replicas`, `storageClassName`, `requests.storage` and other configurations according to the needs of your cluster.
+
+> **Note:**
+>
+> The way TiDB Operator renders resources is different from that of Helm. If you migrate configurations from `tidb-cluster chart values.yaml` to TidbCluster CR, the Pump component might be rolling updated.
+
+### Scheduled Backup
+
+After TiDB Operator is upgraded to v1.1, you can configure scheduled full backup using BackupSchedule CR:
+
+- If the TiDB cluster version < v3.1, refer to [Scheduled backup using mydumper](backup-to-s3.md#scheduled-full-backup-to-s3-compatible-storage)
+- If the TiDB cluster version >= v3.1, refer to [Scheduled backup using BR](backup-to-aws-s3-using-br.md#scheduled-full-backup-to-s3-compatible-storage)
+
+> **Note:**
+>
+> - Currently, with BackupSchedule CR, you can only back up data to S3 and GCS using mydumper, and back up data to S3 using BR. If your scheduled full backup sends data to local Persistent Volume Claim (PVC) before upgrade, it cannot switch to management by CR after upgrade.
+> - If you switch to management by CR, to avoid repeated backup, delete the Cronjob of the former scheduled full backup.
+
+### Drainer
+
+- If Drainer is not deployed before TiDB Operator is upgraded to v1.1, you can deploy Drainer as in [Deploy multiple drainers](maintain-tidb-binlog.md#deploy-multiple-drainers).
+- If Drainer is already deployed before TiDB Operator is upgraded to v1.1, it is recommended to manage Drainer using kubectl.
+
+### TiKV Importer
+
+- If TiKV Importer is not deployed before TiDB Operator is upgraded to v1.1, you can deploy TiKV Importer as in [Deploy TiKV Importer](restore-data-using-tidb-lightning.md#deploy-tikv-importer).
+- If TiKV Importer is already deployed before TiDB Operator is upgraded to v1.1, it is recommended to manage TiKV Importer using kubectl.
```

Comment:
Please translate the [missing part](https://github.com/pingcap/docs-tidb-operator/blob/master/zh/notes-tidb-operator-v1.1.md#%E5%85%B6%E4%BB%96%E7%94%B1-chart-%E7%AE%A1%E7%90%86%E7%9A%84%E7%BB%84%E4%BB%B6%E6%88%96%E8%80%85%E5%8A%9F%E8%83%BD%E5%88%87%E6%8D%A2%E5%88%B0-v11-%E6%94%AF%E6%8C%81%E7%9A%84%E6%96%B9%E5%BC%8F) in the following section.